outputs.tf

```
@@ -19,15 +28,14 @@ output "agent_proxy_container" {
                 awslogs-stream-prefix = "agent-proxy"
             }
         } : null)
-        environment = [
-            {"name": "AEMBIT_AGENT_CONTROLLER", "value": "https://${aws_service_discovery_service.agent-controller.name}.${aws_service_discovery_private_dns_namespace.agent-controller.name}:443"},
-            {"name": "TRUSTED_CA_CERTS", "value": local.all_trusted_ca_certs_base64 },
-            {"name": "AEMBIT_RESOURCE_SET_ID", "value": var.agent_proxy_resource_set_id },
-            {"name": "AEMBIT_AGENT_PROXY_DEPLOYMENT_MODEL", "value": "ecs_fargate" }
-        ]
+        environment = local.agent_proxy_default_environment_variables
     })
 }
 
+output "agent_proxy_default_env_variables" {

```

```suggestion
output "agent_proxy_default_environment" {
```
I'd recommend changing `env_variables` to `environment` to match what is needed by the customer